.github/workflows/ci-build-spm.yml

```diff
@@ -40,9 +48,7 @@ jobs:
       uses: ./.github/actions/setup-node.js
 
     - name: Temporary - only run the build
-      run:
-        cd docker-compose/monitor && make build
-        
+      run:  cd docker-compose/monitor && make build BINARY=${{ matrix.mode.binary }}
```

Comment:
this step is no longer necessary since the test script below calls `make build && make dev` anyway